### PR TITLE
#136 + use config from database and logging

### PR DESCRIPTION
**Pull request by JayeshS**
This branch builds on #136 and adds db-configuration and logging

### CODE REVIEW

File: backend/cms/models/configuration.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Configuration(models.Model):
```
Comment (svenseeberg) [inline backend/cms/models/configuration.py]: @timoludwig do you have any opinion on this model?

File: backend/cms/models/configuration.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    key = models.CharField(max_length=100, unique=True, blank=False)
+    value = models.CharField(max_length=1000, unique=True, blank=False)
```
Comment (svenseeberg) [inline backend/cms/models/configuration.py]: Is the unique flag really required for the value?

Comment (JayeshS) [reply backend/cms/models/configuration.py]: Oops.  :) Will fix.

File: backend/cms/models/configuration.py (hunk 1 above)
Comment (timoludwig) [reply backend/cms/models/configuration.py]: I think it has a very limited use case (as we store all non-sensitive config in the settings.py), but for storing keys it's fine.

**Pull request merged**